awx/ui_next/src/screens/NotificationTemplate/NotificationTemplateList/NotificationTemplateListItem.test.js

```diff
@@ -30,6 +30,7 @@ describe('<NotificationTemplateListItem />', () => {
         <tbody>
           <NotificationTemplateListItem
             template={template}
+            onAddToach={jest.fn()}
```

Comment:
Hey @AlexSCorey was this change supposed to be part of this PR?
```suggestion
            onAddToast={jest.fn()}
```